docs/diagnostic-logs.md

```diff
@@ -0,0 +1,164 @@
+
+    ![Query log in Log Analytics](./images/diagnostic-logs/query-log-in-log-analytics.png)
+ 
+Archive log columns include elements listed in the following table:
+
+Name | Description
+------- | ------- 
+TimeGenerated | Log event time
+Collection | Type of the log event. Allowed values are: `Connection`, `Authorization` and `Throttling`
+OperationName | Operation name of the event
+Location | Location of your Azure SignalR Service
+Level | Log event level
+CallerIpAddress | IP address of your server/client
+Message | Detailed message of log event
+UserId | Identity of the user
+ConnectionId | Identity of the connection
+ConnectionType | Type of the connection. Allowed values are: `Server` \| `Client`. `Server`: connection from server side; `Client`: connection from client side
+TransportType | Transport type of the connection. Allowed values are: `Websockets` \| `ServerSentEvents` \| `LongPolling`
+
+## Trouble Shooting With Diagnostic Logs
+
+To trouble shoot for Azure SignalR Service, you can enable server/client side logs to capture failures. At present, Azure SiganlR Service exposes diagnostic logs, you can also enable logs for service side.
+
+When encountering connection unexpected growing or dropping situation, you can take advantage of diagnostic logs to trouble shoot.
+
+Typical issues are often about connections's unexpected quantity changes. See the next sections about how to trouble shoot.
+
+## Unexpected Connection Dropping
+
+If you encounter unexpected connections drop, firstly enable logs in service, server and client sides.
+
+If a connection disconnects, the diagnostic logs will record this disconnecting event, you will see `ConnectionAborted` or `ConnectionEnded` in `operationName`.
+
+The difference between `ConnectionAborted` and `ConnectionEnded` is that `ConnectionEnded` is a expected disconnecting which is triggered by client or server side. While the `ConnectionAborted` is usually an unexpected connection dropping event, and aborting reason will be provided in `message`.
+
+The abort reasons are listed in the following table:
+
+Reason | Description
+------- | ------- 
+Connection count reaches limit | Connection count reaches limit of your current price tier. Consider scale up service unit
+Application server closed the connection | App server triggers the abortion. It can be considered as an expected abortion
+Connection ping timeout | Usually it is caused by network issue. Consider check your app server's availability from the internet
+Service reloading, please reconnect | Azure SignalR Service is reloading. Azure SignalR support auto-reconnect, you can wait until reconnected or manually reconnect to Azure SignalR Service
+Internal server transient error | Transient error occurs in Azure SignalR Service, should be auto-recovered
+Server connection dropped | Server connection drops with unknown error
```

Comment:
What customer should do for “server connection dropped"?

Reply:
updated